content/applications/finance/accounting/fiscal_localizations/localizations/france.rst

```diff
@@ -59,11 +59,9 @@ In case of non-conformity, your company risks a fine of €7,500.
 
 To get the certification just follow the following steps:
 
-* Install the anti-fraud module fitting your Odoo environment from the
-  *Apps* menu:
-
-  * if you use Odoo Point of Sale: *l10n_fr_pos_cert*: France - VAT Anti-Fraud Certification for Point of Sale (CGI 286 I-3 bis)
-  * in any other case: *l10n_fr_certification*: France - VAT Anti-Fraud Certification (CGI 286 I-3 bis)
+* If you use **Odoo Point of Sale**, install the **France - VAT Anti-Fraud Certification for Point
+  of Sale (CGI 286 I-3 bis)** module by going to :menuselection:`Apps`, removing the *Apps* filter,
```

Comment:
```suggestion
  of Sale (CGI 286 I-3 bis)** module by going to **Apps**, removing the *Apps* filter,
```
A `menuselection` directive for a single step is overkill IMO.

Reply:
the font-weight isn't the same in the render.
Bold is 700 
menuselection is 600

So to me, it would look weird to have an app to open with such a huge font-weight 😜